### PR TITLE
Feature/bench 1354 breadcrumb bug

### PR DESCRIPTION
**Pull request by okyd-capco**
## Ticket

https://ilabs-capco.atlassian.net/browse/BENCH-1354

## Description

The breadcrumb does not navigate to the correct page as indicated in the URL.

## WIP

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).

*PRs that should not be reviewed should be marked as a WIP.

- [ ] This PR is currently a work-in-progress.
- [x] This PR is ready for review.

## Checklist

- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors.
- [x] All existing and new tests are passing.
- [x] My PR meets the acceptance criteria as outlined by my ticket.
- [x] I updated/added relevant documentation and comments (docs start with `///` and comments with `//`).
- [ ] Static analysis (e.g. Snyk) does not report any problems on my PR.

## Screenshots

*Screenshots can be added here.*

### CODE REVIEW

File: src/components/BreadCrumb/index.tsx [hunk 1]
```diff
@@ -39,17 +45,34 @@ const BreadcrumbComponent: FC<RouteComponentProps> = ({
         const isLast = index === pathnames.length;
         const Name = name.toUpperCase();
 
+        const HandleClick = () => {
+          if (
+            pathnames.indexOf(
+              "mobile" ||
+                "frontend" ||
+                "backend" ||
+                "integrtion" ||
```
Comment (jluu-capco) [inline src/components/BreadCrumb/index.tsx:54]: typo

**Review by jluu-capco — COMMENTED** [review]
Just some small comments. Also I notice if you just click on category directly, it always goes to the devops category page.

File: src/components/BreadCrumb/index.tsx (hunk 1 above)
Comment (okyd-capco) [reply src/components/BreadCrumb/index.tsx:54]: Done

File: src/components/BreadCrumb/index.tsx [hunk 2]
```diff
@@ -39,17 +45,34 @@ const BreadcrumbComponent: FC<RouteComponentProps> = ({
...
+                "cloud-&-platforms" ||
+                "quality-engineering" ||
+                "databases" ||
+                "devops"
+            )
+          ) {
+            history.push(`/${pathnames.slice(0, index++).join("/")}`);
+            console.log(routeTo);
```
[the remark this answers is not in the document]
Comment (okyd-capco) [reply src/components/BreadCrumb/index.tsx]: Done

**Review by jluu-capco — APPROVED** [review]

File: src/components/BreadCrumb/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (okyd-capco) [reply src/components/BreadCrumb/index.tsx]: I've removed the console log

**Pull request closed without merging**